e2e/provision/README.md

```diff
@@ -1,5 +1,7 @@
 # Quick Start for GCE
 
+[![Createte a Sandbox in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?repo=nephio-project/test-infra)
```

Comment:
To be executed in codespace, I think that we must check that the scripts are running fine in this environment. We probably need to create a user

Reply:
I've manually tested [here](https://github.com/codespaces/new?repo=635975340&skip_quickstart=true&machine=largePremiumLinux&ref=add_devcontainers_support&geo=UsWest&devcontainer_path=.devcontainer.json), unfortunately not all the GitHub accounts has enabled that flavor.